doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+.. _ug_nrf70_developing_enabling_raw_packet_transmit_feature:
+
+Enabling raw packet transmit feature
+************************************
+
+The raw packet transmit feature for the nrf70 series device is placed under a configuration flag ``CONFIG_NRF700X_RAW_DATA_TX``. The configuration setting can be enabled by enabling the configuration in the ``prj.conf`` of the application attempting to send raw packets.
+
+.. _ug_nrf70_developing_raw_tx_operating_modes:
+
+Raw packet operating modes
+**************************
+
+The nrf70 series device supports raw packet transmission when TX-Injection mode is enabled. A net_mgmt API ``NET_REQUEST_WIFI_MODE`` has been provided to set TX-Injection mode of operation.
+
+The nrf70 series device supports raw packet transmission in the below station operating modes:
+
+* **Station idle mode:** The device is operating as an 802.11 station device and is in idle state (not connected to AP).
+* **Station connected mode:** The device is operating as an 802.11 station device and is in connected state.
+
+To enable raw packet transmission, the nrf70 device mode of operation has to be configured to "Station + TX-Injector mode" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API. When the raw tx mode is enabled, a channel for raw packet transmission also has to be configured mandatorily. This channel will be utilized for raw packet transmission when the station is in disconnected state. When the station is in connected state, the station connected channel will be used for raw packet transmission. Channel configuration cannot be done when station is in connected state. The net_mgmt API ``NET_REQUEST_WIFI_CHANNEL`` can be used to configure the channel of operation.
+
+To disable raw packet transmission, the nrf70 device mode of operation has to be configured to "**station mode**" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring mode setting and channel setting for raw packet transmission via shell commands.
+
+The shell commands and network management APIs to swtich between operational modes and to set channel are provided below
+
+.. list-table:: Wi-Fi raw packet transmission network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example Usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -st
+     - configure interface instance 1 to Station + TX injector mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -s
+     - configure interface instance 1 to pure station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - wifi channel -i<interface instance> <channel number to set>
+     - wifi channel -i1 -c6
+     - set raw transmission channel to 6 for interface 1 to be used in Station idle mode
+
+Raw packet transmit header
```

Comment:
This section should have a more generic title and content. Suggest
"TX Injection mode usage requirements"

Reply:
okay

Reply:
done